### DISCUSSION TITLE
ERROR: No module named 'modules.ui_components'

### DISCUSSION THREAD

**Discussion by Miquel-code** [TheLastBen/fast-stable-diffusion#1246] [General]
Hi guys, please help me, when I ran the model version 1.5 it gave me this error message:

![image](https://user-images.githubusercontent.com/110945646/210488735-e5701202-8c39-494f-b223-e282c427cb49.png)

I've been trying to _"Redownload_the_original_model"_ but it has kept giving me the same message **_" No module named 'modules.ui_components' "._**

I want to make it clear that I'm not an expert to programme. Please help me.

**Comment by tsound97** [comment]
Package and download the latest AUTO1111 webui, overwriting the files in the sd/stable-diffusion-webui folder

**Reply by Miquel-code** [reply]
Thank u, I redownload again stable-diffusion. Of course I saved before my images. Have a nice day.

**Comment by TheLastBen** [comment]
remove/rename the folder sd and use the latest colab to do a clean run

**Reply by Miquel-code** [reply]
Thank you, finally my copy of stable diffusion is working again. Have a nice day.
